chimp.md

```diff
@@ -0,0 +1,213 @@
+- Tornado Cash (Ethereum-based): A mixing protocol that offers transaction anonymity but lacks a fully integrated trading platform like Chimp Exchange.
+- SecretSwap (Secret Network): This DEX offers privacy but is limited to the Secret Network and doesn’t provide the cross-chain functionality that Chimp Exchange is building.
+
+How is Chimp Exchange different?
+- Cross-Chain Privacy Trading: Unlike ZKSwap or SecretSwap, Chimp Exchange is focused on enabling private trading across multiple chains via the Ten Protocol. This cross-chain compatibility, coupled with complete anonymity, sets Chimp apart.
+- End-to-End Encryption: While Tornado Cash and ZKSwap offer forms of transaction mixing, Chimp Exchange integrates full E2EE, ensuring that trades are not only private but also encrypted at every stage.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Akshay Nassa (CEO)
+- Rahul Chaudhary (COO)
+- Saurabh Sharma (CFO)
+- Tejas (CTO)
+- Nidhi (CMO)
+- Prateek (Content Lead) 
+- Jeevesh (Partnership & Community) 
+- Khushi - (HR)
+
+### Contact
+
+- **Contact Name:** Corentin Cadieu
+- **Contact Email:** chimp@0xlabs.tech
+- **Website:** https://chimp.exchange 
+
+### Legal Structure
+
+- **Registered Address:** Add later
+- **Registered Legal Entity:** Name of your registered legal entity, if available. (e.g. Duo Ltd.)
+
+### Team's experience
+
+The Chimp Exchange team brings a wealth of experience in blockchain development, cryptocurrency, and DeFi, led by Akshay Nassa (CEO), a serial entrepreneur with a successful track record in scaling crypto projects like Delta.Exchange and Terablock. Rahul Chaudhary (COO) brings extensive experience in scaling IT and blockchain startups, while Saurabh Sharma (CFO) has expertise in finance management and tokenomics from his work in IT and crypto sectors. Tejas Chitnis (CTO), co-founder of AuroBlocks and former CEO of Cryption Network, brings deep technical expertise in blockchain architecture and smart contract development, making the team well-equipped to build and scale the world’s first privacy-focused DEX.
+
+### Team Code Repos
+
+- https://github.com/orgs/Chimp-Exchange/repositories 
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/akshaynassa/
+- https://ae.linkedin.com/in/rahulchaudhary1
+- https://ae.linkedin.com/in/saurabh-sharma-8540a8282
+- https://www.linkedin.com/in/tejas-chitnis/
+- https://www.linkedin.com/in/nidhi-nassa-030901306/
+- https://www.linkedin.com/in/pratik-bhuyan-0bb42079/
+- https://www.linkedin.com/in/jeeveshkuril/
+- https://www.linkedin.com/in/khushi-arora-84964222b/
+
+
+## Development Status :open_book:
+
+- Whitepaper : WIP
+- System Architecture graphic: https://drive.google.com/file/d/1jCoRayv_8ePR_xET5Nef3N3ge5Thc8el/view?usp=sharing
+- Technical Stack: JavaScript, NodeJS, ReactJS, VueJS, Typescript, Hardhat, Solidity
```

Comment:
You list frontend stuff and solidity here yet in the milestones you are working on a substrate chain. What is the actual tech stack you will be using?